plottable.d.ts

```diff
@@ -1727,6 +1736,19 @@ declare module Plottable {
             };
             fixedWidth(): boolean;
             fixedHeight(): boolean;
+            /**
+             * @return {Component[]} The Components in this Group.
+             */
+            components(): Component[];
+            protected _components(): Component[];
+            /**
+             * Adds a Component to the Group.
+             *
+             * @param {Component} component
+             * @param {boolean} prepend If true, prepends the Component. If false, appends it.
+             */
+            add(component: Component, prepend?: boolean): Group;
```

Comment:
I remember there was talk if we like this siganture.  Ringing the bell that `boolean` parameters are bad